<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assignmentwindow</class>
 <widget class="QMainWindow" name="assignmentwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>assignmentwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="assignmentgrid">
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="lineedit_assignmentname">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font: 13pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
	border-radius: 5px;
	border: 1px solid #632d3b;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_assignmentname">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 16pt &quot;High Tower Text&quot;;
	
	color: rgb(99, 45, 59);
}</string>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5" rowspan="3">
      <widget class="QListWidget" name="listwidget_assignmentlist">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget{
	font: 13pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
	border-radius: 5px;
	border: 1px solid #632d3b;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDateTimeEdit" name="datetime_assignment">
       <property name="styleSheet">
        <string notr="true">QDateTimeEdit{
	font: 13pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
}</string>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_assignmentlist">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 25pt &quot;Copperplate Gothic Light&quot;;
	color: rgb(99, 45, 59);
}</string>
       </property>
       <property name="text">
        <string>Assignment List</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QTextEdit" name="textedit_assignmentdes">
       <property name="styleSheet">
        <string notr="true">QTextEdit{
	font: 13pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
	border-radius: 5px;
	border: 1px solid #632d3b;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_assignmentdes">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 16pt &quot;High Tower Text&quot;;
	color: rgb(99, 45, 59);
}</string>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="label_assignment">
       <property name="styleSheet">
        <string notr="true">QLabel{
	border: none;
	border-bottom-color: 2px solid rgb(99, 45, 59);
	font: 25pt &quot;Copperplate Gothic Light&quot;;
	color: rgb(99, 45, 59);
}</string>
       </property>
       <property name="text">
        <string>Assignment</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="button_assignmentadd">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(99, 45, 59);
	font: 13pt &quot;High Tower Text&quot;;
	background: transparent;
	border: 2px solid #632d3b;
	border-radius: 10px;
	padding: 10px 20;
}
QPushButton:hover{
	background: rgba(255, 255, 255, 0.2)
}
QPushButton:pressed{
	background: rgba(255, 255, 255, 0.1)
}</string>
       </property>
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QPushButton" name="button_assignmentedit">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(99, 45, 59);
	font: 13pt &quot;High Tower Text&quot;;
	background: transparent;
	border: 2px solid #632d3b;
	border-radius: 10px;
	padding: 10px 20;
}
QPushButton:hover{
	background: rgba(255, 255, 255, 0.2)
}
QPushButton:pressed{
	background: rgba(255, 255, 255, 0.1)
}</string>
       </property>
       <property name="text">
        <string>EDIT</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="button_assignmentremove">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(99, 45, 59);
	font: 13pt &quot;High Tower Text&quot;;
	background: transparent;
	border: 2px solid #632d3b;
	border-radius: 10px;
	padding: 10px 20;
}
QPushButton:hover{
	background: rgba(255, 255, 255, 0.2)
}
QPushButton:pressed{
	background: rgba(255, 255, 255, 0.1)
}</string>
       </property>
       <property name="text">
        <string>REMOVE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
